<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sonic Gesture</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icons/thumb_up.png</normaloff>:/icons/icons/thumb_up.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCVImage" name="CVWindow" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pauzeButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/pauze movie</normaloff>:/icons/pauze movie</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="continueButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/play movie</normaloff>:/icons/play movie</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recordButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/record</normaloff>:/icons/record</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="positionSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionOpen_Device_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFinder_View"/>
    <addaction name="actionCapture_View"/>
    <addaction name="actionSource_view"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>&amp;Record</string>
    </property>
    <addaction name="actionRecord_visual_output"/>
    <addaction name="actionRecord_original_movie"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="actionAbout_Sonic_Gesture"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menuRecord"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Video"/>
   <addaction name="actionOpen_Device_2"/>
   <addaction name="separator"/>
   <addaction name="actionSource_view"/>
   <addaction name="actionFinder_View"/>
   <addaction name="actionCapture_View"/>
   <addaction name="separator"/>
   <addaction name="actionRecord_original_movie"/>
   <addaction name="actionRecord_visual_output"/>
   <addaction name="separator"/>
   <addaction name="actionFullscreen"/>
  </widget>
  <action name="actionOpen_Video">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/film</normaloff>:/icons/film</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Video...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOpen_Device_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/camera</normaloff>:/icons/camera</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Device...</string>
   </property>
  </action>
  <action name="actionFinder_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/finder view</normaloff>:/icons/finder view</iconset>
   </property>
   <property name="text">
    <string>Finder view</string>
   </property>
  </action>
  <action name="actionCapture_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/capture view</normaloff>:/icons/capture view</iconset>
   </property>
   <property name="text">
    <string>Capture view</string>
   </property>
  </action>
  <action name="actionRecord_original_movie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/record input</normaloff>:/icons/record input</iconset>
   </property>
   <property name="text">
    <string>Record input</string>
   </property>
  </action>
  <action name="actionRecord_visual_output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/record output</normaloff>:/icons/record output</iconset>
   </property>
   <property name="text">
    <string>Record output</string>
   </property>
  </action>
  <action name="actionSource_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/normal view</normaloff>:/icons/normal view</iconset>
   </property>
   <property name="text">
    <string>Source view</string>
   </property>
  </action>
  <action name="actionAbout_Sonic_Gesture">
   <property name="text">
    <string>About Sonic Gesture</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/fullscreen</normaloff>:/icons/fullscreen</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCVImage</class>
   <extends>QWidget</extends>
   <header>qcvimage.h</header>
   <container>1</container>
   <slots>
    <slot>about()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen_Video</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Device_2</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFinder_View</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>finderView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRecord_original_movie</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>recordInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRecord_visual_output</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>recordOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSource_view</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>sourceView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauzeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pauze()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>continueButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changePosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCapture_View</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>captureView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFullscreen</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>fullscreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openVideo()</slot>
  <slot>openDevice()</slot>
  <slot>finderView()</slot>
  <slot>captureView()</slot>
  <slot>sourceView()</slot>
  <slot>recordInput()</slot>
  <slot>recordOutput()</slot>
  <slot>pauze()</slot>
  <slot>play()</slot>
  <slot>changePosition()</slot>
  <slot>record()</slot>
  <slot>fullscreen()</slot>
  <slot>about()</slot>
 </slots>
</ui>
